### DISCUSSION TITLE
Question 35: What would be the best way for other collections to be notified of host/parasite data within EMu? [RB - INS]

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#538] [Q&A] [answered]
Description:  

While generating a workflow for host/parasite data, we wondered if there was any way to automate this process within EMu. Is there a method that could be used to notify other collections that an internal relationship between specimens needs resolving aside from Tasks? Our [currently developing workflow](https://docs.google.com/document/d/1juY_R0bX6EWNFP7o9JP6CNQqzX_JaHWQL8US6N5EAD8/edit?usp=sharing) leaves the Mammal host records without a link to the soon to be entered specimens, kind of hanging unresolved. This’ll become more common as time goes on, so we wanted to check options with the Trinity.

**Comment by fmjjones** [comment, marked as answer]
See the [Relationship workflow](https://docs.google.com/document/d/190msXiIYgui4zReB4O8-IQXq4Gue0GqQZU3wMCZG_jo/edit#heading=h.l6ge4s76ok6f) doc for more information.
